Bio/PDB/StructureBuilder.py

```diff
@@ -118,7 +118,7 @@ def init_residue(self, resname, field, resseq, icode):
         if field != " ":
             if field == "H":
                 # The hetero field consists of H_ + the residue name (e.g. H_FUC)
-                field = "H_" + resname
+                field = ("H_" + resname).replace(" ", "")
```

Comment:
I would rather use `str.strip` here than `replace`. More explicit. You can call it on the residue name only.

Reply:
Can I use `str.lstrip()`? It will be even more explicit. Or, should I simply use `str.strip()`?

Reply:
Do you know that spaces always come the left? :)